modules/tinymce/src/plugins/table/test/ts/browser/TabKeyNavigationTest.ts

```diff
@@ -42,4 +44,26 @@ describe('browser.tinymce.plugins.table.TabKeyNavigationTest', () => {
       '</td></tr><tr><td>&nbsp;</td><td>&nbsp;</td></tr></tbody></table><p>x</p>'
     );
   });
+  it('TBA: Fire TableModified event when row are added via Tab key', () => {
+    const editor = hook.editor();
+
+    const events: Array<EditorEvent<TableModifiedEvent>> = [];
+    const logEvent = (event: EditorEvent<TableModifiedEvent>) => {
+      events.push(event);
+    };
+    editor.on('TableModified', logEvent);
+
+    editor.setContent('<table><tbody><tr><td>A1</td><td>A2</td></tr><tr><td>B1</td><td>B2</td></tr></tbody></table><p>x</p>');
+
+    LegacyUnit.setSelection(editor, 'tr:nth-child(2) td:nth-child(2)', 0);
+    assert.lengthOf(events, 0);
+    editor.fire('keydown', { keyCode: Keys.tab() } as KeyboardEvent);
```

Comment:
I realise you've done what's in the test above, but I believe you can use `TinyContentActions.keystroke` here.

Something like
```suggestion
    TinyContentActions.keystroke(editor, Keys.tab());
```

Reply:
Yeah, using `fire` was an older way of doing things. We should try and use the relevant keyboard test APIs we have now instead, as that's much closer to a real keyboard event.